pom.xml

```diff
@@ -922,6 +925,10 @@
             <maxmem>1024m</maxmem>
             <compilerArgs>
               <arg>-Xlint:none</arg>
+              <arg>--add-exports=java.base/jdk.internal.access=ALL-UNNAMED</arg>
+              <arg>--add-exports=java.base/jdk.internal=ALL-UNNAMED</arg>
+              <arg>--add-exports=java.base/jdk.internal.misc=ALL-UNNAMED</arg>
+              <arg>--add-exports=java.management/sun.management=ALL-UNNAMED</arg>
```

Comment:
are the `add-exports` lines needed for java 11 support? i remember these from getting java17 runtime working.

Reply:
> the classes in alluxio.concurrent.jsr were added as part of https://github.com/Alluxio/alluxio/pull/8827/files
> 
> 1. double checking that removed classes are identical to the ones provided by java 11?
> 2. i see some additional exclusion lines for alluxio.concurrent.jsr in pom.xml, remove those too?

1. The ones provided by Java 11 should act the same according to the JDK docs.
2. @dbw9580 we don't need `alluxio.concurrent.jsr` any more, and we can remove the related declarations in `pom.xml`.

Reply:
> are the `add-exports` lines needed for java 11 support? i remember these from getting java17 runtime working.

We need these lines as the classes in package `jdk.internal` are invisible in Java 11.